sphinx/cmdline.py

```diff
@@ -121,7 +121,7 @@ def main(argv):
                       help='show version information and exit')
 
     group = parser.add_option_group('General options')
-    group.add_option('-b', metavar='BUILDER', dest='builder', default='html',
+    group.add_option('-b', action='append', metavar='BUILDER', dest='builder', default=[],
```

Comment:
This would enable a target in my makefile that looks like:

    $(SPHINXBUILD) -b html -b text -b linkcheck ...